presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -648,6 +654,11 @@ public static boolean isSpatialJoinEnabled(Session session)
         return session.getSystemProperty(SPATIAL_JOIN, Boolean.class);
     }
 
+    public static String getSpatialPartitioningTableName(Session session)
```

Comment:
Don't return null. Make it `Optional<String>`, and return `Optional.fromNullable(...)`